scraper/dev_journal_artem.txt

```diff
@@ -0,0 +1,30 @@
+DATE:   03.05.23
+Could not scrape the data for some of the fields in newYorkTimes spider
+Tried to use if statements so they would check for different layouts 
+and scrape the data for those fields. Didn't work even though the xpath command 
+extracted whatever I wanted to get. Will ask for help tomorrow in class
+
+Another thing that was literally was driving me crazy was extracting images
+for one layout. I could see the element img in the inspect element but I couldn't 
+get it with xpath and it was returning svg that I couldn't find on the page. Some 
+magic outside of Hogwarts, idk.
+
+DATE:   04.05.23
+Aardhyn helped me to fix it. It was easier than I thought.... 
+just a syntax error
+Made a decent progress for the newYorkTimes spider. Reduced the amount of null fields
+from 37 to 20. The rest of the layouts were irrelevant to our website, so we decided
+to leave them as it is.
+
+DATE:   07.05.23
+Finished polishing New York Times spider and finished doing Fox News spider
+Took me some time to figure out why some of the conditions were not triggered 
+in my if statement. Was a bit painful to run spider after every change so
+I could check if it functions as intended.
+Need to sort out categories for BBC and CNN spiders 
+
```

Comment:
Great that your recording your progress! Something I need to do 🤔.

Can this be part of a Wiki page on Scraping, rather than stored in the actual repository.

Reply:
Yes, we can do that perhaps. When I was talking to Elise and Adon, they said that it's absolutely fine to keep `dev_notes/journal` in the repo (in the directory where you work). However, I agree that it's a better idea to keep it in the _wiki_. Let's do that instead